src/main/java/org/jenkinsci/plugins/workflow/actions/ErrorAction.java

```diff
@@ -62,6 +65,10 @@ public ErrorAction(@NonNull Throwable error) {
             }
         }
         this.error = error;
+        String id = findId(error, new HashSet<>());
+        if (id == null && error != null) {
+            error.addSuppressed(new ErrorId());
+        }
```

Comment:
If you are looking at stack traces, things look a bit weird, but maybe it's not that bad?

`Functions.printThrowable` makes things pretty confusing:

```
Also:   org.jenkinsci.plugins.workflow.actions.ErrorAction$ErrorId: ebb451ba-8b2f-47de-93e5-071e84186409
Also:   	Also:   org.jenkinsci.plugins.workflow.actions.ErrorAction$ErrorId: 40a72e1e-e303-4272-b603-d93f6053e65d
hudson.AbortException: script returned exit code 1
hudson.AbortException: script returned exit code 1
	at org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:667)
	at org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:613)
	at org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:557)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

I think it is easier to understand the structure with `Throwable.printStackTrace` in this case:

```
hudson.AbortException: script returned exit code 1
	at org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:667)
	at org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:613)
	at org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:557)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
	Suppressed: org.jenkinsci.plugins.workflow.actions.ErrorAction$ErrorId: cbaf6c15-6f81-4eab-ae4a-a8123230d190
	Suppressed: hudson.AbortException: script returned exit code 1
		... 9 more
		Suppressed: org.jenkinsci.plugins.workflow.actions.ErrorAction$ErrorId: 02e6416b-c7b5-42a5-9502-5d40f20357f2
```

Reply:
`printThrowable` formatting around suppressed exceptions could likely be improved.